bzt/modules/java.py

```diff
@@ -461,3 +461,14 @@ def _parse_mirrors(self):
             links.append(default_link)
         self.log.debug('Total mirrors: %d', len(links))
         return links
+
+
+class JavaEnv(object):
```

Comment:
If only Gatling uses it, can we skip introducing new entity for now? Looks like overkill...